### PR TITLE
CET-4647 [Adobe] Backstage Plugin - Progress Reports Grouped by Level…

### PR DESCRIPTION
**Pull request by voytekgar**
… returns 400

## Change description

> Description here

## Type of change

- [ ] Bug fix (fixes an issue)
- [ ] New feature (adds functionality)
- [ ] Improvement (improves codebase without changing functionality)

### Development

- [ ] The changelog has been updated as appropriate
- [ ] Lint rules pass locally
- [ ] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### Code review

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

**Review by cristinabuenahora — COMMENTED** [review]
You'll want to bump the package version too

**Comment by voytekgar** [comment]
> You'll want to bump the package version too

@cristinabuenahora thx, will keep it in mind

**Review by FoodProduct — COMMENTED** [review]
+1 changelog and then LGTM

**Comment by voytekgar** [comment]
@cristinabuenahora @FoodProduct 
>  can you update the changelog

done

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 2.4.2
+
+- Block not supported grouping by level option for historical report.
```
[the remark this answers is not in the document]
Comment (voytekgar) [reply CHANGELOG.md]: thx

**Review by FoodProduct — APPROVED** [review]

**Pull request merged**